dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -306,10 +306,10 @@ class RubricsControllerTest < ActionController::TestCase
   end
 
   test "run ai evaluations for user calls EvaluateRubricJob" do
-    create :user_level, user: @student, script: @rubric.lesson.script, level: @level
+    create :user_level, user: @student, script: @script_level.script, level: @level
```

Comment:
weird that these are not the same! I'll make a note to follow up on that

Reply:
oh woops -- I think this might be an overzealous find/replace. They should be the same